app/javascript/dashboard/routes/dashboard/settings/inbox/Settings.vue

```diff
@@ -316,6 +316,22 @@
           >
             <woot-code :script="inbox.hmac_token"></woot-code>
           </settings-section>
+          <settings-section
+            :title="$t('INBOX_MGMT.SETTINGS_POPUP.HMAC_MANDATORY_VERIFICATION')"
+            :sub-title="
+              $t('INBOX_MGMT.SETTINGS_POPUP.HMAC_MANDATORY_DESCRIPTION')
+            "
+          >
+            <div class="enter-to-send--checkbox">
+              <input
+                id="hmacMandatory"
+                v-model="hmacMandatory"
+                type="checkbox"
+                @change="handleHmacFlag"
+              />
+              <label for="hmacMandatory"> Enable </label>
```

Comment:
The label should be language-supported JSON files.

Reply:
Yes, use i18n here

Reply:
Resolved [here](https://github.com/chatwoot/chatwoot/pull/3137/commits/859f77cc9b40f3105fea23f278a528874021e257)